frontends/web/src/components/aopp/aopp.tsx

```diff
@@ -226,11 +226,11 @@ export const Aopp = () => {
           </p>
           <Field>
             <Label>{t('aopp.labelAddress')}</Label>
-            <CopyableInput alignLeft flexibleHeight value={aopp.address} />
+            <CopyableInput alignLeft flexibleHeight value={aopp.address} dataTestId={'aopp-address'}/>
```

Comment:
Altough camelCasing is needed in JSX, React treats all data-* and aria-* attributes as passthrough attributes.

And in the rest of the codebase we have started using data-testid (not camelCased).

```suggestion
            <CopyableInput alignLeft flexibleHeight value={aopp.address} data-testid="aopp-address" />
```